app/models/constants/involved_civilian.rb

```diff
@@ -62,5 +64,7 @@ module InvolvedCivilian
     FIELDS_NOT_COLLECTED_IF_FLED = [
       :highest_charge, :crime_qualifier, :confirmed_armed, :age, :gender, :race, :injured
     ].freeze
+
+    K12_TYPE = ['A student', 'A school employee', 'Neither'].freeze
   end
```

Comment:
I would also add a new label in `CUSTOM_LABELS_FOR_REVIEW` above, perhaps something like:
```
  k12_type: "Civilian's role on K-12 campus"
```